jsonnet/kube-prometheus/kube-state-metrics/kube-state-metrics.libsonnet

```diff
@@ -1,313 +1,53 @@
-local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
+local ksm = (import 'kube-state-metrics/kube-state-metrics.libsonnet') +
+            {
+              name:: 'kube-state-metrics',
+              namespace:: 'monitoring',
+              version:: '1.9.4',  //$._config.versions.kubeStateMetrics,
+              image:: 'quay.io/coreos/kube-state-metrics',  //$._config.imageRepos.kubeStateMetrics + ':v' + $._config.versions.kubeStateMetrics,
```

Comment:
Image needs to include the version, you can declare a local bar before name like this:

local cfg = self,

Then you can have image be what you have now “+ cfg.version”.

Reply:
fixed.